File: Facebook.NET - VB/Contracts/GraphAPI.AccountData.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class AccountData
        Implements GraphApi.IObject
            <DataMember(Name:="id")>
            Public Property ID() As String Implements IObject.ID
            <DataMember(Name:="name")>
            Public Property Name() As String Implements IObject.Name
            <DataMember(Name:="category")>
            Public Property Category() As String
            <DataMember(Name:="access_token")>
            Public Property AccessToken() As String
            <DataMember(Name:="perms")>
            Public Property Permissions() As String()
        End Class
    End Class
End Namespace

File: Facebook.NET/Contracts/GraphAPI.AccountData.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed class AccountData : GraphApi.IObject
        {
            [DataMember(Name = "id")]
            public string ID { get; set; }

            [DataMember(Name = "name")]
            public string Name { get; set; }
            
            [DataMember(Name = "category")]
            public string Category { get; set; }
            
            [DataMember(Name = "access_token")]
            public string AccessToken { get; set; }
            
            [DataMember(Name = "perms")]
            public string[] Permissions { get; set; }
        }
    }
}